sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/spatial/GeometryDeserializer.java

```diff
@@ -20,7 +20,7 @@
 /**
  * Deserializes a JSON object into a {@link Geometry}.
  */
-final class GeometryDeserializer extends JsonDeserializer<Geometry> {
+public final class GeometryDeserializer extends JsonDeserializer<Geometry> {
```

Comment:
This is fine for testing this, in the future when this work is merged into Azure Core it'll become package private again.

Reply:
Right, I think it doesn't hurt to public the serializer before we move back Azure Core.